king_phisher/client/application.py

```diff
@@ -101,14 +101,24 @@ class KingPhisherClientApplication(_Gtk_Application):
 	# pylint: disable=too-many-public-methods
 	__gsignals__ = {
 		'campaign-changed': (GObject.SIGNAL_RUN_FIRST, None, (str,)),
+		'campaign-created': (GObject.SIGNAL_RUN_FIRST, None, (str,)),
+		'campaign-deleted': (GObject.SIGNAL_RUN_FIRST, None, (str,)),
 		'campaign-set': (GObject.SIGNAL_RUN_FIRST, None, (str,)),
+		'config-load': (GObject.SIGNAL_RUN_LAST, None, ()),
 		'config-save': (GObject.SIGNAL_RUN_LAST, None, ()),
+		'credential-deleted': (GObject.SIGNAL_RUN_FIRST, None, (str,)),
 		'exit': (GObject.SIGNAL_RUN_LAST, None, ()),
 		'exit-confirm': (GObject.SIGNAL_RUN_LAST, None, ()),
+		'message-deleted': (GObject.SIGNAL_RUN_FIRST, None, (str,)),
+		'message-sent': (GObject.SIGNAL_RUN_LAST, None, (str, str)),
```

Comment:
Since this is a dummy signal used as a hook point and the handler doesn't exist / is just pass it should probably be `GObject.SIGNAL_RUN_FIRST`.